docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -580,59 +776,115 @@ public int Shape
+   _textVisual.PointSize = 18;
+   _textVisual.MultiLine = true;
+   _textVisual.TextColor = Color.Blue;
+   _textVisual.FontFamily = "Arial";
+   // Text centered horizontally within its area
+   _textVisual.HorizontalAlignment = HorizontalAlignment.Center;
+   // Text aligned to the top edge of its area
+   _textVisual.VerticalAlignment = VerticalAlignment.Top;
+   // Parent's reference point in the top-left corner
+   _textVisual.Origin = Visual.AlignType.TopBegin;
+   // Visual's reference point - top-center
+   _textVisual.AnchorPoint = Visual.AlignType.TopCenter;
+   // Visual's reference point shifted to the center (horizontally)
+   _textVisual.RelativePosition = new RelativeVector2(0.5f, 0);
+
+   PropertyMap _fontStyle = new PropertyMap();
+   _fontStyle.Add("weight", new PropertyValue("extrabold"));
+   _fontStyle.Add("width", new PropertyValue("ultracondensed"));
+   _fontStyle.Add("slant", new PropertyValue("italic"));
+   _textVisual.FontStyle = _fontStyle;
+
+   PropertyMap _underline = new PropertyMap();
+   _underline.Insert("enable", new PropertyValue("true"));
+   _underline.Insert("color", new PropertyValue(Color.Black));
+   _underline.Insert("height", new PropertyValue("10"));
+   _textVisual.Underline = _underline;
+
+   PropertyMap _shadow = new PropertyMap();
+   _shadow.Insert("color", new PropertyValue(new Color(0.4f, 0.4f, 1.0f, 1.0f)));
+   _shadow.Insert("offset", new PropertyValue(new Vector2(5,5)));
+   _shadow.Insert("blurRadius", new PropertyValue(3));
+   _textVisual.Shadow = _shadow;
 
-```
-PropertyMap textVisual = new PropertyMap();
-textVisual.Add(Visual.Property.Type, new PropertyValue((int)Visual.Type.Text))
-          .Add(TextVisualProperty.Text, new PropertyValue(_name))
-          .Add(TextVisualProperty.TextColor, new PropertyValue(Color.White))
-          .Add(TextVisualProperty.PointSize, new PropertyValue(7))
-          .Add(TextVisualProperty.HorizontalAlignment, new PropertyValue("CENTER"))
-          .Add(TextVisualProperty.VerticalAlignment, new PropertyValue("CENTER"));
-_textVisual = VisualFactory.Get().CreateVisual(textVisual);
-RegisterVisual(TextVisualPropertyIndex, _textVisual);
+   _visualView.AddVisual("sample_text", _textVisual);
+   ```
 
-/// Set the text visual depth index
-_textVisual.DepthIndex = TextVisualPropertyIndex;
-```
 
-<a name="wireframevisual"></a>
-## Wireframe Visual
+**Figure: TextVisual from the code above (_visualView background color set to gray)**
+
+![Text visual](media/visuals/TextVisualExample.png)
+
 
-The wireframe visual renders a wireframe around a control's quad. It is mainly used for debugging and is the visual that replaces all other visuals when **Visual Debug Rendering** is switched on. To switch **Visual Debug Rendering** on, set the `DALI_DEBUG_RENDERING` environment variable to 1 before launching the application.
 
-**Figure: Wireframe visual**
 
-![Wireframe visual](media/wireframe-visual.png)
 
 ## Related Information
```

Comment:
## Related information

Reply:
I'm not sure what should I change here

Reply:
We are using sentence case in all the documents, so kindly update. 

I to i

Reply:
I'm a little bit confused, because I see such a heading in all already merged sections of nui guide

Reply:
I do understand, please refer to https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization

Thanks,